corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_buttons.scss

```diff
@@ -35,3 +35,20 @@
 .btn-outline-danger:hover {
   color: #ffffff;
 }
+
+.btn-default {
+  @include button-variant(white, $call-to-action-mid,  $call-to-action-mid);
+}
+
+.btn-group-xs > .btn, .btn-xs {
+  padding: 1px 5px;
+  font-size: 12px;
+  line-height: 1.5;
+  border-radius: 3px;
+  }
```

Comment:
What is this set of styles needed for?

Reply:
`btn-xs` no longer exists for bootstrap 5 so this style recreates it